### PR TITLE
Build script for building JS SDK Docs

### PR DESCRIPTION
**Pull request by arv**
This script npm installs documentationjs and runs it on noms.js and
puts the output in $WORKSPACE/build

It is intended to be used with Jenkins.

### CODE REVIEW

File: tools/build-js-documentation.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    assert workspace
+
+    noms_dir = os.path.join(workspace, 'src/github.com/attic-labs/noms')
+    noms_js_dir = os.path.join(noms_dir, 'js/noms')
+    with open(os.path.join(noms_js_dir, 'package.json')) as pkg:
+        data = json.load(pkg)
+
+    version = data['version']
+
+    os.chdir(workspace)
```
Comment (kalman) [inline tools/build-js-documentation.py]: noms_js_dir?

Comment (arv) [inline tools/build-js-documentation.py]: Why?

**Comment by kalman** [comment]
lgtm modulo 1 question

I also suggest that you install pylint and run it against all python scripts, fix any sensible errors.

File: tools/build-js-documentation.py (hunk 1 above)
Comment (kalman) [inline tools/build-js-documentation.py]: You don't want to run it relative to the js directory?

Comment (arv) [inline tools/build-js-documentation.py]: To be honest it doesn't really matter. Maybe it is better to be in a dir that already have a `node_modules` in it but then there might be some conflict on the version of `documentation` we get.

Comment (kalman) [inline tools/build-js-documentation.py]: Hm, I assumed given src/js already had node_modules and so forth you might want to keep everything in there. But I see it doesn't especially matter.

Comment (kalman) [inline tools/build-js-documentation.py]: Conflicting comments :-) SGTM.

**Pull request merged**